plugins/gatsby-theme-iterative-docs/src/components/Documentation/Markdown/Main/styles.module.css

```diff
@@ -273,6 +273,53 @@
     margin-left: 20px;
     margin-right: 10px;
   }
+
+  .yt-embed-wrapper {
```

Comment:
I'm not sure if this design is good enough for a first iteration 🤔 

![image](https://user-images.githubusercontent.com/43496356/152210976-7eb931bd-c331-4677-ae06-4cc6bd1afc31.png)

Reply:
This looks great to me! This happens on hover, right?

Reply:
Yes!

https://user-images.githubusercontent.com/43496356/152249881-212056ca-1460-4ad7-97fd-3eae85226479.mov